ngraph/test/type_prop/if.cpp

```diff
@@ -0,0 +1,300 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "gtest/gtest.h"
```

Comment:
Please don't add new eval tests. Add new tests to https://github.com/openvinotoolkit/openvino/tree/master/docs/template_plugin/tests/functional/op_reference

Reply:
will be done as a separate story (62130)